utils/pvacviz/app.py

```diff
@@ -35,11 +35,20 @@ def do_GET(self):
 
 def main():
     os.chdir(CLIENTDIR)
+    parser = argparse.ArgumentParser(description='parse pvacviz arguments')
+    parser.add_argument('--ip_address', help='IP address for the viz HTTP server to bind')
```

Comment:
For other commands we've been using dash-separated naming of optional arguments. So this should be changed to `--ip-address`.